tasks/SticklerForRules.tex

```diff
@@ -0,0 +1,46 @@
+\section{Stickler for the Rules [Party Host]}
+The robot has to enforce house rules set by the homeowner.
+
+
+\subsection{Main Goal}
+The robot has to recognize when party guests are breaking rules and politely ask them to stop.
+
+\noindent\textbf{Reward:} 500pts (100pts per recognized rule breaking).
```

Comment:
Spelling: rule **breaker**

I think finding 5 misbehaving people for 100pts each is too much. I'd place the same 5 but only two are making a mess, so 250 each. Also you have 3 being nice, so there is space for false positives the robot must avoid

Reply:
I think rule breaking is fine here, both work. 
Addressed the points in commit.